### PR TITLE
[FIX] Lion Dance Mask missing in-game Bumpkin

### LINKED ISSUE #2085: [BUG] Lion Dance Mask missing on teeny player Bumpkin
## Summary of Bug

If you equipped the Lion Dance Mask SFT on a Bumpkin wearing a Reindeer Suite the players in-game teeny tiny Bumpkin will not wear the mask while the profile image of the Bumpkin in the HUD is wearing the mask.

## Steps to Reproduce

1. Equip the Reindeer Suite on your Bumpkin
2. Equip the Lion Dance Mask on your Bumpkin
3. Click `Save to blockchain` button and approve transaction
4. Wait 30-60 seconds after the transaction is processed
5. Load up or refresh Sunflower Land in your browser

## Expected Behavior

Tiny Bumpkin should be wearing the Lion Dance Mask like it's larger counterpart in the HUD

## Actual Behavior

Tiny Bumpkin is just chilling without the Lion Dance Mask looking like an adorable reindeer

## Screenshots

|  Big screenshot                                                                                                                                           |   Little screenshots                                                                                                                                                                                                                                                                                        |
| ------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| ![Screenshot from 2023-01-23 20 04 41](https://user-images.githubusercontent.com/801056/214211254-7d8a7bb9-9aea-42a1-a962-1d36084fc82d.png) | ![Screenshot from 2023-01-23 20 05 47](https://user-images.githubusercontent.com/801056/214211256-4baceef7-c27b-44c7-af0e-91cb90f392f0.png) ![83_1_5_14_19_22_44_84_28_0_0_0_69x100](https://user-images.githubusercontent.com/801056/214211253-4a2504f1-da7f-4e60-9c71-98d24591f2f2.png) |


**Device Information (please complete the following information):**

- Device: Dell XPS 13 Plus, Dell XPS 15 9510, Pixel 7 Pro
- OS: Linux (Dell laptops), Android
- Browser/App: Chrome (Linux), MetaMask (Android)
- Version [e.g. 22]: v1.0.72

FIEXED #2095

### PR DESCRIPTION
**Pull request by therocco**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

The `Lion Dance Mask` when worn only shows up in the players HUD avatar. The players in-game Bumpkin didn't get to wear the fancy hat as well.

Fixes #2085 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

For this change I took the mask off the NPC that wears it on the island and carefully created a sprite out of it that matched the hair and hat sprites. I then attached it to the Bumpkin body parts and tested it via a local dev server by swapping it out for regular hair (I couldn't figure out how to get the hat on my Testnet Bumpkin's NFT)

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

I couldn't find where the tests were in relation to Bumpkin costumes but I am more than happy to update the existing tests if there are any.

https://user-images.githubusercontent.com/801056/214700488-42471838-4e16-4f3e-a77b-d934ef0e79f3.mp4

Below you can see the screenshots I took while making sure the hat fit on all the different NPC models, even the goblins even though the ears poke out of the sides of the hat which I suspect is what it would look like in real life

Screenshot from 2023-01-25 16-35-36

Screenshot from 2023-01-25 16-35-58

Screenshot from 2023-01-25 16-36-35

Screenshot from 2023-01-25 16-36-46

Screenshot from 2023-01-25 16-37-00

Not only did I make sure they lined up with all the sprites currently in the game but taking inspiration from the original NPC that was wearing the hat I made the bottom line on the hat a black band with 75% opacity applied to it so it keeps the spirit of the original design hat by looking like it casts a shadow on the players neck and chest area giving it a as much depth as you can get out of 19px tall characters.

Screenshot from 2023-01-25 16 48 39

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by mGranTY** [comment]
Hello @therocco i know you had all the trouble to make the sprite, i love this PR but can we use the official sprite made by our bumpkin designer @LittleEins ?

File: src/features/island/bumpkin/components/DynamicMiniNFT.tsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ import snowman from "assets/npc-layers/snowman-onesie.png";
 import sharkOnesie from "assets/npc-layers/shark-onesie.png";
 import reindeerSuit from "assets/npc-layers/reindeer_suit.png";
 import reindeerAntlers from "assets/npc-layers/reindeer_antlers.png";
+import lionDanceMask from "assets/npc-layers/lion_dance_mask.png";
```
Comment (mGranTY) [inline src/features/island/bumpkin/components/DynamicMiniNFT.tsx:46]: feel free to download the original spritesheet on my now closed PR https://github.com/sunflower-land/sunflower-land/pull/2096

**Review by mGranTY — CHANGES_REQUESTED** [review]

File: src/features/island/bumpkin/components/DynamicMiniNFT.tsx (hunk 1 above)
Comment (therocco) [reply src/features/island/bumpkin/components/DynamicMiniNFT.tsx:46]: Sure thing! It will be checked in and pushed to this PR in a few minutes.

**Review by mGranTY — APPROVED** [review]
LGTM!

**Pull request merged**